mobile/src/actions/settings.js

```diff
@@ -71,9 +73,17 @@ export const registerDevice = () => async (dispatch, getState) => {
   dispatch(checkURL(getState().mobile.settings.serverUrl))
   initClient(getState().mobile.settings.serverUrl, onRegister(dispatch), device)
   await cozy.client.authorize().then(async ({ client }) => {
+    dispatch(unrevokeClient())
     dispatch(setClient(client))
     await cozy.client.offline.replicateFromCozy('io.cozy.files')
-    cozy.client.offline.startRepeatedReplication('io.cozy.files', 15)
+    const options = {
+      onError: (err) => {
```

Comment:
Replication is an event emitter maybe it ok with onError : https://github.com/pouchdb/pouchdb/blob/master/docs/_includes/api/replication.html#L9